src/compute/src/logging/compute.rs

```diff
@@ -255,7 +257,8 @@ pub fn construct<A: Allocate>(
                                         Datum::String(&dataflow.to_string()),
                                         Datum::String(&source_id.to_string()),
                                         Datum::Int64(worker as i64),
-                                        Datum::Int64(logical as i64),
+                                        // TODO: Convert to MzTimestamp.
+                                        Datum::Int64(u64::from(logical) as i64),
```

Comment:
This `as` seems as bad as the ones above? Should it also be `try_into()`?

Reply:
Going away in the next commit.